ccm_web/client/src/pages/BulkSectionCreate.tsx

```diff
@@ -312,7 +312,7 @@ function BulkSectionCreate (props: CCMComponentProps): JSX.Element {
 
   const parseFile = (file: File): void => {
     file.text().then(t => {
-      let lines = t.split(/[\r\n]+/).map(line => { return line.trim() })
```

Comment:
This was eating up consecutive newlines so any empty lines were getting ignored, which was fine if there were no errors, but if there was an error the line numbers might be incorrect.